actions/onboarding.js

```diff
@@ -11,10 +11,12 @@ export const COMPLETED_KEYS = Object.freeze({
 
 export const SET_STEP = 'SET_STEP';
 export function setStep(step) {
-  SecureStore.setItemAsync(ONBOARDING_STEP, step.toString());
-  return {
-    type: SET_STEP,
-    step,
+  return (dispatch) => {
+    SecureStore.setItemAsync(ONBOARDING_STEP, step.toString());
```

Comment:
Is this a number or a string?

Reply:
Ah okay. I would highly recommend keeping them as a string representing the step. That way if the order or steps change it would still work just fine.